translations/catroweb.sq_AL.yml

```diff
@@ -184,6 +184,7 @@ programs:
   showTranslation: Shfaq përkthimin 
   hideTranslation: Fshihe përkthimin
   translatedByLine: Perkthyer nga %provider% prej %sourceLanguage% në %targetLanguage%
+  stealProject: 'Vidh projektin'
```

Comment:
Just as a note: It is only necessary to add new strings to catroweb.en.yml
All other translations are handled over Crowdin and therefore your translations might be lost on the next update.